tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.collections)
```

Comment:
If you did want to wait for the first collection to be displayed, your condition could be `len(self.collections) > 0 and self.collections[0].is_displayed`